src/Test/Perf/util/test_util.csx

```diff
@@ -255,28 +262,42 @@ void Log(string info)
     }
 }
 
-/// Logs the result of a finished process
+// Logs an error message.
+//
+// The actual implementation of this method may change depending on
+// if the script is being run standalone or through the test runner.
+void LogError(string info)
```

Comment:
It turns out the .Flush is not necessary.  I'll remove it.  Any objections/thoughts on Log vs LogError with the .Flush call removed?